app/views/users/index.html.erb

```diff
@@ -5,9 +5,9 @@
 		<% @users.each do |user| %>
 		<div class="col-md-4">
 			<%= link_to(user) do %>
-			<h1><%= user.userName %></h1>
+			<h3 class="text-center userIndex"><%= user.userName.downcase %></h3>
```

Comment:
ideally define a convention for naming CSS attributes.  I see spinalcase `text-center` and camel case `userIndex`.